### DISCUSSION TITLE
在输入完整的地址可以不使用 baseUrl 吗

### DISCUSSION THREAD

**Discussion by raymondZhong** [alovajs/alova#312] [Q&A]
### 请详细描述bug

在输入完整的地址可以不使用 baseUrl 吗
![image](https://github.com/alovajs/alova/assets/1297621/d1ed6325-21d8-4f61-acf9-a29ac0a6c822)

### 期望的表现

在输入完整的适合可以，可以不使用 baseUrl 吗。

### alova 版本

2.18.0

**Comment by JOU-amjs** [comment]
问题请在discussion进行。

目前不支持你的这种场景，两种方法可以解决，如下：
1. 创建一个不设置baseURL的新alova实例来发请求
2. 在这个报错的method上将baseURL设置为空
```js
const getter = alovaInstance.Get(...);
getter.baseURL = '';

// 再发请求
const res = await getter;
```
